source/guides/module_guides/bgtm.md

```diff
@@ -58,194 +60,310 @@ In terms of class structure we recommend the following (more detail below):
 
-Thus, if you want to allow other modules to form ordering relationships with your module, you should ensure that your main class(es) explicitly _contain_ any subordinate classes they declare. For more information and context, you can see [the Containment page of the Puppet reference manual](/puppet/latest/reference/lang_containment.html).
+#### Containment and anchoring
 
-In Puppet 3.4.0 / Puppet Enterprise 3.2 and later, you can contain classes by using [the `contain` function](/puppet/latest/reference/function.html#contain) on them. To support versions prior to 3.4.0 / PE 3.2, you must use the **anchor pattern** to hold those classes in place. See below for an example of anchoring. (*Note: anchoring requires [puppetlabs-stdlib](http://forge.puppetlabs.com/puppetlabs/stdlib)*.)
+To allow other modules to form ordering relationships with your module, ensure that your main classes explicitly _contain_ any subordinate classes they declare.
 
-Two resources to anchor things to:
+Classes do not _automatically_ contain the classes they declare. This is because classes can be declared in several places via `include` and similar functions. To contain classes, use [the `contain` function](/puppet/latest/function.html#contain). For more information and context about containment, see [the containment docs](/puppet/latest/lang_containment.html).
 
-~~~ ruby
-    anchor { 'module::begin': }
-    anchor { 'module::end' }
-~~~
-
-The anchoring:
-
-~~~ ruby
-    Anchor['module::begin'] ->
-      Class['module::install'] ->
-      Class['module::config']  ->
-      Class['module::service'] ->
-    Anchor['module::end']
-~~~
-
-This enforces ordering, so `::install` will happen before `::config`, and
-`::config` before `::service`.
-
-For a real world example of anchoring in action, please see Hunter's [puppet-wordpress](https://github.com/hunner/puppet-wordpress/blob/master/manifests/init.pp) module.
+Containment is supported in Puppet 3.4 and later. To support versions prior to Puppet 3.4 (or Puppet Enterprise 3.2), you must use the [anchor pattern][anchor] to hold those classes in place. Anchoring requires [puppetlabs-stdlib](http://forge.puppetlabs.com/puppetlabs/stdlib).
 
 ### 2d: Dependencies
 
-If your module's functionality depends on another module, then you must list these dependencies and include them directly, rather than indirectly hoping they are included in the catalog.
+If your module's functionality depends on another module, then you must list these dependencies and include them directly.
 
 This means you must `include x` in your `init.pp` to ensure the dependency is included in the catalog. You must also add the dependency to the module's [metadata.json](/guides/style_guide.html#module-metadata) and `.fixtures.yml`. (`.fixtures.yml` is a file used exclusively by rspec to pull in dependencies required to successfully run unit tests.)
```

Comment:
Minor nit: it should be included in the *main class*, which is a more specific location within init.pp.